include/wx/versioninfo.h

```diff
@@ -23,15 +23,26 @@ class wxVersionInfo
                   int major = 0,
                   int minor = 0,
                   int micro = 0,
+                  int tweak = 0,
                   const wxString& description = wxString(),
                   const wxString& copyright = wxString())
         : m_name(name)
         , m_description(description)
         , m_copyright(copyright)
+        , m_major(major)
+        , m_minor(minor)
+        , m_micro(micro)
+        , m_tweak(tweak)
+    {
+    }
+
+    // This constructor exists for backward compatibility (before the tweak
+    // part was added).
+    wxVersionInfo(const wxString& name, int major, int minor, int micro,
+                  const wxString& description,
+                  const wxString& copyright = wxString())
+        : wxVersionInfo(name, major, minor, micro, 0, description, copyright)
```

Comment:
Oh, damn, sorry, I missed this during the review, but we still support C++98 and so can't use delegating constructors, so you'll have to duplicate the above lines here or add `Init()` function, as done in other places.